packages/docs/components/header/components/LanguageDropdown.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <va-dropdown class="language-dropdown" :offset="[0, 25]" fixed>
+    <va-icon slot="anchor" :class="['flag-icon flag-icon-large', flagIconClass(currentLanguage)]" />
+    <div class="language-dropdown__content pl-4 pr-4 pt-2 pb-2">
+      <div
+        v-for="(option, id) in options"
+        :key="id"
+        class="language-dropdown__item row align--center pt-1 pb-1 mt-2 mb-2"
+        :class="{ active: option.code === currentLanguage }"
+        @click="setLanguage(option.code)"
+      >
+        <va-icon :class="['flag-icon flag-icon-small', flagIconClass(option.code)]" />
+        <span class="dropdown-item__text">{{ $t(`language.${option.name}`) }}</span>
+      </div>
+    </div>
+  </va-dropdown>
+</template>
+
+<script>
+import { Component, Vue } from 'vue-property-decorator'
+import VaIcon from 'vuestic-ui/src/components/vuestic-components/va-icon/VaIcon.vue'
+import VaDropdown from 'vuestic-ui/src/components/vuestic-components/va-dropdown/VaDropdown.vue'
+import VaButton from 'vuestic-ui/src/components/vuestic-components/va-button/VaButton.vue'
+@Component({
+  name: 'language-dropdown',
+  components: {
+    VaIcon,
+    VaDropdown,
+    VaButton,
+  },
+  props: {
+    options: {
+      type: Array,
+      default: () => [
+        {
+          code: 'gb',
+          name: 'english',
+        },
+        {
+          code: 'es',
+          name: 'spanish',
+        },
+        {
+          code: 'br',
+          name: 'brazilian_portuguese',
+        },
+        {
+          code: 'cn',
+          name: 'simplified_chinese',
+        },
+      ],
+    },
+  },
+})
+export default class LanguageDropdown extends Vue {
+  beforeMount () {
+    const { locales } = this.$root.$i18n
```

Comment:
Should be moved to service/singleton.
We might have pages without header.

Reply:
extracted to a separate plugin